eventually/authentication/models.py

```diff
@@ -50,6 +66,18 @@ def get_by_id(user_id):
             return None
 
     @staticmethod
+    def get_by_email(user_email):
+        """
+        Returns user by email
+        :param user_email: email by which we need to find the user
```

Comment:
indicate the type of the function parameter with sphinx.

Reply:
Fixed.